contrib/labstack/echo.v4/option.go

```diff
@@ -8,18 +8,24 @@ package echo
 import (
 	"math"
 
+	"github.com/labstack/echo/v4"
+
 	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
 )
 
 type config struct {
 	serviceName   string
 	analyticsRate float64
 	noDebugStack  bool
+	skipFunc      SkipFunction
 }
 
 // Option represents an option that can be passed to Middleware.
 type Option func(*config)
 
+// SkipFunction determines if tracing will be skipped for a request.
+type SkipFunction func(c echo.Context) bool
```

Comment:
With the new naming of `WithIgnoreRequest` perhaps these "skipFunction" related names should be changed to match? Perhaps `IgnoreRequestFunc`? (and similar changes for other related names)

Reply:
Hi @ajgajg1134. Sure thing, I've changed all references to "skip". The reason I had used this term is because "skip" seems to be the way this type of behaviour is described within the [rest of the echo framework middleware](https://github.com/labstack/echo/blob/master/middleware/middleware.go#L15). Also it is not really dealing with a "request", but an echo context (which allows us to base decisions on more than what a plain old http.Request provides).

Having said all of that, what actually matters to me is the functionality, and that works fine regardless of the name. One side fits in with Echo, the other fits in with the rest of the middleware within _this_ repo (which is probably more important in terms of consistency).

Naming is never easy, and I appreciate your taking the time to get this right!